src/highlevel.jl

```diff
@@ -346,7 +372,7 @@ macro roc(ex...)
     args = call.args[2:end]
 
     code = quote end
-    macro_kwargs, compiler_kwargs, call_kwargs, signal_kwargs, kernel_kwargs = split_kwargs(kwargs)
+    device_kw, macro_kwargs, compiler_kwargs, call_kwargs, signal_kwargs, kernel_kwargs = split_kwargs(kwargs)
```

Comment:
```suggestion
    device_kwargs, macro_kwargs, compiler_kwargs, call_kwargs, signal_kwargs, kernel_kwargs = split_kwargs(kwargs)
```

Reply:
Alternatively, you're welcome to rename these all to `xyz_kw` if you think that's easier to read.